vllm/worker/tpu_worker.py

```diff
@@ -130,7 +130,7 @@ def determine_num_available_blocks(self) -> Tuple[int, int]:
         # Calculate the TPU KV cache size based on profiling.
         usable_memory_size = int(total_memory_size *
                                  self.cache_config.gpu_memory_utilization)
-        tpu_kv_cache_bytes = max(usable_memory_size - profiled, 0)
+        tpu_kv_cache_bytes = max(usable_memory_size - profiled, 0) // 4
```

Comment:
why do "// 4"?

Reply:
NVM, I will remove it. This is only used to understand the cost of the torch.permute on the pre-allocated kv cache buffer. Anyway after the new pallas attn kernel is finished, the torch.permute will disappear.